src/Trixi.jl

```diff
@@ -48,6 +48,11 @@ using StaticArrays: MVector, MArray, SMatrix
 using StrideArrays: PtrArray, StrideArray, StaticInt
 using StructArrays: StructArrays, StructArray
 using TimerOutputs: TimerOutputs, @notimeit, TimerOutput, print_timer, reset_timer!
+
+using Triangulate
+using TriplotBase
+using TriplotRecipes: DGTriPseudocolor
```

Comment:
Could you please restrict the `using` statements to make clear which types/functions you import and use in Trixi.jl?